cmd/ddev/cmd/commands_test.go

```diff
@@ -499,6 +499,9 @@ func TestNpmYarnCommands(t *testing.T) {
 		out, err := exec.RunHostCommand(DdevBin, "npm", "install", "--no-audit")
 		assert.NoError(err)
 		assert.Contains(out, "up to date in", "d='%s', npm install has wrong output; output='%s'", d, out)
+		out, err := exec.RunHostCommand(DdevBin, "npx", "-c", "'echo hello'")
```

Comment:
Please make sure you can run this test manually. Try

```suggestion
		out, err := exec.RunHostCommand(DdevBin, "npx", "-c", "echo hello")
```